content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -287,7 +287,7 @@ The table below describes LEDs meaning and functionalities.
 
 ### Out-Of-The-Box Experience
 
-***Before following this tutorial, please update your Portenta X8 to the latest version. Check [this section](#portenta-x8-os-image-update) to learn how to do it.***
+***If you would like to have your Portenta X8 with the latest OS version. Check [this section](#portenta-x8-os-image-update) to learn how to do it.***
```

Comment:
I wouldn't say "If you would like to have...", because some of the functionalities are now included in the previous OS version, e.g. Cloud, new out of the box etc.

Reply:
It is now rephrased to be more "global", taking into account the details as mentioned. Let me know if it expresses better now.